File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int freq[]=new int[nums.length];
        int j=0;
        for(int i=0;i<nums.length-1;i++){
            if(nums[i]!=nums[i+1]){
            freq[j++]=nums[i];
            }
        }
        freq[j++]=nums[nums.length-1];
        for(int i=0;i<nums.length;i++){
                nums[i]=freq[i];
            
        }
        return j;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i,j;
        for(i=0,j=1;j<nums.size();j++){
            if(nums[i]!=nums[j]) 
                nums[++i]=nums[j];
        }
        return i+1;
    }
};